resources/lib/metadata.py

```diff
@@ -173,6 +173,8 @@ def get_playcount(self, api_data):
                 total = self._resumepoints.get_total(asset_id)
                 if position and total and position > total - SECONDS_MARGIN:
                     playcount = 1
+                elif position and total and position < SECONDS_MARGIN:
```

Comment:
This doesn't seem right. We don't know the playcount is 0 in this case, and it is better to not set it and fall back on what Kodi already knows. (unknown will be zero anyway)

Reply:
This is needed to overwrite the Kodi watch status so it matches VRT NU resumepoint positions.

Reply:
But if you watch it first completely once, and then watch it again (even in the first 30 seconds), playcount would be 0. I don't think we need to explicitly reset playcount in the listitem in this case.

Reply:
Indeed, that was the purpose. But I have removed this because I'm not sure what is best for the user keeping in mind that VRT NU resumepoints breaks the watched/unwatched context menu.